src/react/views/admin/articles/edit.js

```diff
@@ -89,7 +98,7 @@ export default class ArticleEditPage extends Component {
                                   <div className="row"><br /></div>
                                   <div className="row">
                                       <ArticleLayout>
-                                          <ArticleEdit article={article} />  
+                                          <ArticleEdit ref={(el) => this.articleEdit = el} article={article} />
```

Comment:
"React supports a special attribute that you can attach to any component. The `ref` attribute takes a callback function, and the callback will be executed immediately after the component is mounted or unmounted."

https://facebook.github.io/react/docs/refs-and-the-dom.html